app.py

```diff
@@ -155,12 +155,17 @@ def upload_strategy():
     bucket_name = app.config["S3_BUCKET"]
 
     # path: e.g. s3://com34156-strategies/{user_id}/strategy_num/{strategy_name}.py
-    response = s3_client.list_objects_v2(
-        Bucket=bucket_name, Prefix=userid
+    
+    conn = rds.get_connection()
+    strategies = pd.read_sql(
```

Comment:
this is a bit anti-pattern, but I think it is fine. 
ideally please use cursor for this. 
also, please rename `strategies` to `current_max_strategy_id`